packages/components/addon/components/hds/form/utils/setAriaDescribedBy.js

```diff
@@ -1,4 +1,4 @@
-export function getAriaDescribedBy(element) {
+export function setAriaDescribedBy(element) {
```

Comment:
I have a couple of alternatives that may simplify the code further (`element.descriptors` is already an array, so no need to re-build it via a `forEach` and `push` on a new array).

```
export function setAriaDescribedBy(element) {
  let ariaDescribedBy = element.descriptors || [];

  // append @extraAriaDescribedBy arg, if provided
  if (element.args.extraAriaDescribedBy) {
    element.descriptors.push(element.args.extraAriaDescribedBy);
  }

  if (ariaDescribedBy.length) {
    element.ariaDescribedBy = ariaDescribedBy.join(' ');
  }
}
```

or if you want to completely get rid of the temp local variable `ariaDescribedBy` you could use directly the `element.descriptors` array itself:

```
export function setAriaDescribedBy(element) {
  // append @extraAriaDescribedBy arg, if provided
  if (element.args.extraAriaDescribedBy) {
    element.descriptors.push(element.args.extraAriaDescribedBy);
  }

  if (element.descriptors.length) {
    element.ariaDescribedBy = element.descriptors.join(' ');
  }
}
```

Reply:
Indeed, now that we fixed the async issue that made me keep that counterintuitive iteration, we can simply this code further and I like your suggestion above